src/ray/common/id.cc

```diff
@@ -70,11 +70,10 @@ void FillNil(T *data) {
 }
 
 WorkerID ComputeDriverIdFromJob(const JobID &job_id) {
-  std::vector<uint8_t> data(WorkerID::Size(), 0);
+  std::string data(WorkerID::Size(), '\0');
```

Comment:
Q: cpp strings are not immutable right? i.e., fill_N replaces chars in place

Reply:
Right it’s just like vector of char